library/swift/EngineBuilder.swift

```diff
@@ -27,6 +27,7 @@ open class EngineBuilder: NSObject {
   private var enableBrotli: Bool = false
   private var enableInterfaceBinding: Bool = false
   private var enforceTrustChainVerification: Bool = true
+  private var enablePlatformCertificateValidation: Bool = true
```

Comment:
Should this default to `false` until it's been vetted in a production experiment?

```suggestion
  private var enablePlatformCertificateValidation: Bool = false
```